### DISCUSSION TITLE
Display Image

### DISCUSSION THREAD

**Discussion by HMM-MK** [filamentphp/filament#8357] [General]
FileUpload::make('doctor_photo') 
                     ->label('Doctor Photo') 
                     ->disk('local') 
                     ->directory('doctor_photo') 
                     ->visibility('private') 
                         ->getUploadedFileNameForStorageUsing( 
                             fn (TemporaryUploadedFile $file): string => (string) str($file->getClientOriginalName()) 
                                 ->prepend('doctor-')
                    )

As the above upload, I store the doctor photo and saved the doctor_photo (doctor_photo/file path) in DB.

And I use ImageColumn::make('doctor_photo') to display but not show the any, image src not have src path.

How to make that, Please help me?

**Comment by leandrocfe** [comment]
[Private images - Discord topic](https://discordapp.com/channels/883083792112300104/1145723690839973908/1145725698229665886)
